### ISSUE TITLE
[Paladin] [Skill] One-Handed Weapon Specialization not working

### ISSUE THREAD

**Issue by SFCDaniel** [dalaranwow/dalaran-wow#5924]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
after skilling 3/3 "One-Handed Weapon Specialization" from the Protection Tree of the Paladin the damage of Consecration and Judgements didn't increase while having a one handed Sword or Mace equiped

**Expected behaviour**: Tell us what should happen instead
the damage of all damage Abilities and Spells should increase by 10% 

**Steps to reproduce the problem**:

1. test damage(Judgment/Consecration) without "One-Handed Weapon Specialization" and a one handed weapon equiped
2. allocate 3 Points into the "One-Handed Weapon Specialization" Skill
3. see that the damage doesn't get altered by 10% but stays the same 

**Include proofs for this behaviour**
https://wowwiki.fandom.com/wiki/One-Handed_Weapon_Specialization_(paladin_talent)
increases "ALL damage" not only Melee damage

https://www.wowhead.com/forums&topic=157560
" It used to be more common in TBC than it is now, but I have actually still seen tanks who pass up the 5 points on one handed weapon spec. 1HWS is our #1 BIGGEST threat talent, but some people chose to pass over it. Why? They don't read the tooltip properly and assume it only affects melee damage aka white damage aka not holy damage. So they pass up on this talent thinking it doesn't affect their threat much because they think it doesn't do anything to holy damage when in fact it is the complete opposite."

https://www.wowhead.com/forums&topic=1181
in TBC the Wording (which is the same in WOTLK!) changed from "Increases the damage you deal with one-handed melee weapons by 10%" to "increases all damage you deal when a one handed melee weapon is equipped by 10%" 

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Skipslegdays** [comment]
Can confirm that the talent is still broken. Ran some numbers today.
Unarmed Consecration: 381 per tick
Consecration with Wrathful Mace (144 ap) equipped: 388 per tick

Honestly surprised that no one's assigned to this issue for 9 months.

**Comment by Chenocide** [comment]
> Can confirm that the talent is still broken. Ran some numbers today.
> Unarmed Consecration: 381 per tick
> Consecration with Wrathful Mace (144 ap) equipped: 388 per tick
> 
> Honestly surprised that no one's assigned to this issue for 9 months.

Dalaran-wow doesn't really assign issues to anyone. They used to do it when they had some freelance devs working on bug fixing but otherwise it's uncommon.
